kontan-server/src/services/OfficeService.ts

```diff
@@ -20,47 +20,66 @@ export interface UpcomingPresenceDto {
   users: User[];
```

Comment:
A thought: 

In this file on line 101, there's a fine reduce. 
I'm thinking like this, if you sign up with any other office than Helsingborg (if you don't have a tag), your tag will be saved as "NO_TAG" 

That way we can set another status, and show a different result on "Today" in the app, if there's anyone without a tag planning to come to Hbg. 

So for instance: 

```
export type Status = 'INBOUND' | 'OUTBOUND' | 'PLANNED' | 'PLANNED_NO_TAG';
```
And then in reduce:

```
const getPlannedStatusOrWhateverThisShouldBeCalled = (tag: string) => {
  return tag === "NO_TAG" ? "PLANNED_NO_TAG" : "PLANNED";
}

const userMap = [...physicallyCheckedIn, ...plannedForToday].reduce<
      Record<User['slackUserId'], InboundDto>
    >((acc, user) => {
      
      return {
        ...acc,
        [user.slackUserId]: {
          ...user,
          status: checkedInSet.has(user.slackUserId)
            ? presence.office[user.tag].status
            : getPlannedStatusOrWhateverThisShouldBeCalled(user.tag),
        },
      };
    }, {});
```

This way you can determine if you're going to render "Not checked in yet" or just their name, in case they don't have a tag. 

As a suggestion :)